src/v2/controllers/FormController.js

```diff
@@ -0,0 +1,41 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+define([
+  'util/BaseLoginController',
+  '../views/FormView',
+  '../models/FormModel',
+  '../../views/shared/FooterWithBackLink'
+],
+function (
+  BaseLoginController,
+  FormView,
+  FormModel,
+  FooterWithBackLink
+) {
+  return BaseLoginController.extend({
+    className: 'enroll-user',
```

Comment:
nit: `enroll-user` is probably too specific ?

Reply:
hah, this needs to be fixed